### DISCUSSION TITLE
Pop certain screens from stack

### DISCUSSION THREAD

**Discussion by dineshvg** [adrielcafe/voyager#85] [Q&A] [answered]
There is a `popUnitl` which pops everything till the desired screen. 
If there was a method like `popOnly(screen)` that can iterate through the stack and remove a particular screen if it exists, that would be handy in some conditions. 

Is there a way to do this already?

**Comment by Syer10** [comment]
I would suggest creating an issue with this request

**Reply by dineshvg** [reply, marked as answer]
Added issue: https://github.com/adrielcafe/voyager/issues/299
